kollider-price/src/convert.rs

```diff
@@ -0,0 +1,23 @@
+use super::price_feed::error::KolliderPriceFeedError;
+use super::price_feed::KolliderPriceTicker;
+
+use shared::{
+    payload::{
+        ExchangeIdRaw, InstrumentIdRaw, KolliderBtcUsdSwapPricePayload, PriceMessagePayload,
+        PriceRatioRaw,
+    },
+    time::TimeStamp,
+};
+
+impl TryFrom<KolliderPriceTicker> for KolliderBtcUsdSwapPricePayload {
+    type Error = KolliderPriceFeedError;
+    fn try_from(value: KolliderPriceTicker) -> Result<Self, Self::Error> {
+        Ok(KolliderBtcUsdSwapPricePayload(PriceMessagePayload {
+            exchange: ExchangeIdRaw::from("Kollider"), // FIXME
```

Comment:
Akin to what is done [here](https://github.com/GaloyMoney/stablesats-rs/blob/1edc01cc9beaec918a1d432c135701e28acd1c7d/shared/src/payload/constants.rs#L1), you can create a constant for the exchange ID.

It appears that Kollider has its unique instrument ID, i.e. `BTCUSD.PERP` but do not return a timestamp with the response from the tickers channel. Unless @bodymindarts has a different opinion, creating a timestamp on the fly as the ticks are received is fine